### PR TITLE
Implement tests for UDTF compositions

### PR DESCRIPTION
**Pull request by pearu**
Requires https://github.com/omnisci/omniscidb-internal/pull/5440

### CODE REVIEW

File: rbc/tests/test_omnisci_udtf.py [hunk 1]
```diff
@@ -1,49 +1,140 @@
...
-        'CREATE TABLE IF NOT EXISTS {table_name} ({table_defn});'
-        .format(**locals()))
+    for o in omnisci_fixture(globals()):
+        define(o)
+        yield o
+
+
+def define(omnisci):
+
+    T = ['int64', 'int32', 'int16', 'int8', 'float64', 'float32']
```
Comment (guilhermeleobas) [inline rbc/tests/test_omnisci_udtf.py:15]: I suppose you don't really need to test all combinations of types given you're testing if function combination works in omniscidb.

File: rbc/tests/test_omnisci_udtf.py [hunk 2]
```diff
@@ -1,49 +1,140 @@
...
+                if j1 == j2:
+                    out[j1] = col1[i1] * col2[i2]
+                    rowid_out[j1] = i1
+                    break
+        return len(col1)
+
+
+@pytest.mark.parametrize("kind", ['i8', 'i4', 'i2', 'i1', 'f8', 'f4'])
+def test_composition(omnisci, kind):
+    omnisci.require_version((5, 6), 'Requires omniscidb-internal PR 5440')
```
Comment (guilhermeleobas) [inline rbc/tests/test_omnisci_udtf.py:64]:
```suggestion
    omnisci.require_version((5, 7), 'Requires omniscidb-internal PR 5440')
```
The omniscidb-internal version is 5.7, but this always confuses me.

**Review by guilhermeleobas — APPROVED** [review]
Thanks @pearu. Code looks good to me, just a couple of comments.

File: rbc/tests/test_omnisci_udtf.py (hunk 1 above)
Comment (pearu) [reply rbc/tests/test_omnisci_udtf.py:15]: true, but one never knows what will break :) Skipping some types will give only a minor decrease in testing time.

File: rbc/tests/test_omnisci_udtf.py (hunk 2 above)
Comment (pearu) [reply rbc/tests/test_omnisci_udtf.py:64]: yeah, it seems the omniscidb-internal master is always ahead by 0.1 of an upcoming release version.

**Pull request merged**